spec/src/main/asciidoc/chapters/language/expressions.adoc

```diff
@@ -243,9 +255,31 @@ The `#type()#` function accepts an expression assigned an entity type, and, when
 An implementation of Jakarta Query targeting clients written in Java assigns the type `java.lang.Class<? extends E>` where `E` is the Java class mapping the entity type assigned to the argument expression, and an entity type expression evaluates to an instance of `Class` representing the Java class which maps the entity type of the record to which the argument expression evaluates.
 
 [[aggregate-functions]]
-==== Aggregate functions
+==== Aggregate functions calls
+
+An _aggregate function call_ may only occur in the `select` or `having` clause of a query involving <<aggregation,aggregation>>.
+Such a clause operates on a list of nested result lists, as specified above in <<projection-and-aggregation>> and <<restriction-and-aggregation>>.
+An aggregate function call is evaluated in the context of such a nested list.
+
+This specification defines the standard aggregate functions listed in the table below.
+
+NOTE: Functions highlighted in yellow belong to the extended language and are not required for an implementation of the core language.
+
+[cols="15,15,15,15,~"]
+|===
+| Function name | Parameters | Parameter types | Type | Semantics
+
+| `count` | 1 | Any type | Long integer | The number of nested list elements for which the argument expression evaluates to a non-null value
+| `#min#` | 1 | Any ordered type `O` | `O` | The largest non-null value of the argument expression over all nested list elements
+| `#max#` | 1 | Any ordered type `O` | `O` | The largest non-null value of the argument expression over all nested list elements
+| `#sum#` | 1 | Any numeric type `N` | `N` | The sum of non-null values of the argument expression over all nested list elements
+| `#avg#` | 1 | Any numeric type `N` | `N` | The average of non-null values of the argument expression over all nested list elements
```

Comment:
Just in case a reader wants clarification on which type of average it is
```suggestion
| `#avg#` | 1 | Any numeric type `N` | `N` | The average (arithmetic mean) of non-null values of the argument expression over all nested list elements
```